scripts/lib/api/processHtml.test.ts

```diff
@@ -247,6 +249,26 @@ test("replaceSourceLinksWithGitHub()", () => {
   );
 });
 
+test("convertRubricsToHeaders()", () => {
```

Comment:
Excellent test. This is so helpful! Thank you for finding all the known edge cases.